sigs.yml

```
@@ -643,7 +643,7 @@
     - name: Daniel Gomez Blanco
       github: danielgblanco
   - name: Security
-    meeting: Weekly on Wednesday at 08:30 AM PT
+    meeting: Weekly on Wednesday at 14:00 UTC
```

```suggestion
    meeting: Bi-weekly on Wednesday at 14:00 UTC
```

as per https://github.com/open-telemetry/sig-security/commit/1b42b98f71221646be1ed514d8ab1770713745a6